train:
  ros__parameters:
    environment: 'TwoCar'  # CarTrack, CarBeat, TwoCar, MultiAgent
    track: 'narrow_multi_track' # track_1, track_2, track_3, multi_track, multi_track_testing
    car_name: 'f2tenth'
    algorithm: 'TD3'
    g: 1
    buffer_size: 1000000
    max_steps_exploration: 1000
    observation_mode: 'lidar_only'
    max_steps_training: 1000000
    is_1d: True
    latent_size: 10
    number_eval_episodes: 3
    max_steps: 250